audius-cli

```diff
@@ -682,12 +672,33 @@ def update_chainspec(ctx):
         ctx.obj["manifests_path"]
         / "discovery-provider"
         / "chain"
-        / "spec_template.json"
+        / f"{network}_spec_template.json"
     )
+
     spec_output = (
         ctx.obj["manifests_path"] / "discovery-provider" / "chain" / "spec.json"
     )
     spec_data = json.load(open(spec_input))
+    
+    prev_network_id = None
+    network_id = spec_data["params"]["networkID"]
+
+    if os.path.isfile(spec_output):
+        prev_spec_data = json.load(open(spec_output))
+        prev_network_id = prev_spec_data["params"]["networkID"]
+        if prev_network_id != network_id:
+            print("Resetting chain")
+            subprocess.run(
+                [
+                    "docker",
+                    "rm",
+                    "--force",
+                    "chain",
+                ],
+            )
+            clique_db_path = "/var/k8s/discovery-provider-chain/db/clique/"
+            subprocess.run(['sudo', 'rm', '-rf', clique_db_path])
+
```

Comment:
Is manual removal the only way here?

Reply:
yea i think it's the only way. i tried doing `docker rm -f -v chain` but that didn't work. i'm guessing cuz it needs sudo?